portability-spi-cloud/src/main/java/org/dataportabilityproject/spi/cloud/storage/JobStore.java

```diff
@@ -72,7 +72,7 @@
    */
   UUID findFirst(JobAuthorization.State jobState);
 
-  default <T extends DataModel> void create(UUID jobId, T model) {
+  default <T extends DataModel> void create(UUID jobId, T model) throws IOException {
```

Comment:
Looks like the travis build is failing since some of the classes that use this dont catch the exception: 
> flickr/src/main/java/org/dataportabilityproject/datatransfer/flickr/photos/FlickrPhotosImporter.java:89: error: unreported exception IOException; must be caught or declared to be thrown
>       jobStore.create(jobId, tempPhotosData);

I think this also has to do with some of the discussion in #176, so tagging that explicitly.